docs/src/ComponentsPage.js

```diff
@@ -733,6 +733,12 @@ const ComponentsPage = React.createClass({
                   <p>Set the <code>header</code> prop to create a structured item, with a heading and a body area.</p>
                   <ReactPlayground codeText={Samples.ListGroupHeader} />
 
+                  <h3><Anchor id="listgroup-with-custom-children">With custom component children</Anchor></h3>
```

Comment:
Fair enough. I might reword the copy to say something like:

>
When using `ListGroupItem`s directly, `ListGroup` looks at whether the items have `href` or `onClick` props to determine which DOM elements to emit. However, with custom item components ...